cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -32,6 +32,15 @@
 #define CDC_MAX_PACKET_SIZE USB_MAX_EP0_SIZE
 #endif
 
+/*
+ * The value USB_CDC_TRANSMIT_TIMEOUT is defined in terms of HAL_GetTick() units.
+ * Typically it is 1ms value. The timeout determines when we would consider the
+ * host "too slow" and threat the USB CDC port as disconnected.
+ */
+#ifndef
+#define USB_CDC_TRANSMIT_TIMEOUT 3 

```

Comment:
Astyle failed due to space at the end
```suggestion
#define USB_CDC_TRANSMIT_TIMEOUT 3
```

Reply:
fixed.